### PR TITLE
Clean up socket logic.

### PR DESCRIPTION
**Pull request by jjskuld**
## Short Description:
Moving some of the socket logic from CLI to Logic

### CODE REVIEW

File: PoGo.NecroBot.CLI/PoGo.NecroBot.CLI.csproj [hunk 1]
```diff
@@ -143,28 +143,19 @@
       <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.10.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="SuperSocket.Common, Version=1.6.6.1, Culture=neutral, PublicKeyToken=6c80000676988ebb, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\SuperSocket.1.6.6.1\lib\net45\SuperSocket.Common.dll</HintPath>
-      <Private>True</Private>
+      <HintPath>..\packages\SuperSocket.1.6.6.1\lib\net45\SuperSocket.Common.dll</HintPath>
```
Comment (CDAGaming) [inline PoGo.NecroBot.CLI/PoGo.NecroBot.CLI.csproj]: Change ..\packages to $(SolutionDir)\packages

File: PoGo.NecroBot.CLI/PoGo.NecroBot.CLI.csproj [hunk 2]
```diff
@@ -143,28 +143,19 @@
...
     </Reference>
     <Reference Include="SuperSocket.SocketService, Version=1.6.6.1, Culture=neutral, PublicKeyToken=6c80000676988ebb, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\SuperSocket.Engine.1.6.6.1\lib\net45\SuperSocket.SocketService.exe</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="SuperSocket.WebSocket, Version=1.6.6.1, Culture=neutral, PublicKeyToken=6c80000676988ebb, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\SuperSocket.WebSocket.1.6.6.1\lib\net45\SuperSocket.WebSocket.dll</HintPath>
-      <Private>True</Private>
+      <HintPath>..\packages\SuperSocket.Engine.1.6.6.1\lib\net45\SuperSocket.SocketService.exe</HintPath>
     </Reference>
```
Comment (CDAGaming) [inline PoGo.NecroBot.CLI/PoGo.NecroBot.CLI.csproj]: Change ..\packages to $(SolutionDir)\packages

**Comment by CDAGaming** [comment]
These are just a few but there are some others

**Comment by jjskuld** [comment]
@CDAGaming Thanks for catching those!

**Comment by CDAGaming** [comment]
Yup, you got them, rest of this PR seems all good to me, and ill add it to my personal build to test later

File: PoGo.NecroBot.CLI/PoGo.NecroBot.CLI.csproj [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Lord-Haji) [inline PoGo.NecroBot.CLI/PoGo.NecroBot.CLI.csproj]: Invisible character

**Comment by CDAGaming** [comment]
Will Merge with my PR very shortly

**Pull request merged**